static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -152,7 +135,8 @@ define([
             return 'media:' + eventName;
         }).forEach(function(playerEvent) {
             player.on(playerEvent, function(_, mediaEvent) {
-                ga(gaTracker + '.send', 'event', buildGoogleAnalyticsEvent(mediaEvent, events, canonicalUrl));
+                ga(gaTracker + '.send', 'event', gaHelper.buildGoogleAnalyticsEvent(mediaEvent, events, canonicalUrl,
+                    'guardian-videojs', getGoogleAnalyticsEventAction, mediaEvent.mediaId));
```

Comment:
The hosted pages use atoms too iirc, worth checking with them how they want to track it. My vote would be to use atom id everywhere for simplicity.

cc @guardian/labs-beta @guardian/labs-beta)